crates/contract/src/call.rs

```diff
@@ -190,36 +189,38 @@ impl CallDecoder for () {
 /// [sol]: alloy_sol_types::sol
 #[derive(Clone)]
 #[must_use = "call builders do nothing unless you `.call`, `.send`, or `.await` them"]
-pub struct CallBuilder<P, D> {
+pub struct CallBuilder<N: Network, T, P, D> {
```

Comment:
I think we want N to default to Ethereum everywhere? @gakonst @prestwich

Reply:
this requires moving N to the end, so would need another PR to alloy-sol-macro. up to you

Reply:
that's fine

Reply:
let's do as follow-up, smol breakage not blocking us from merging & can pass on to contributor

Reply:
https://github.com/alloy-rs/alloy/issues/263